src/js/blocks/tabs/block.js

```diff
@@ -0,0 +1,120 @@
+const { __ } = wp.i18n;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['wp' is not defined. (no-undef)](https://app.codacy.com/manual/uriweb/uri-component-library/pullRequest?prid=4757242)